lesson3.js

```diff
@@ -1,15 +1,13 @@
 const ul = document.getElementById('js-list');
-for(let i=1; i<=2; i++) {
+const imgbox = ['bookmark.png','message.png'];
+const fragment = document.createDocumentFragment();
+for(let i=1; i<=imgbox.length; i++) {
     const li = document.createElement('li');
     const a = document.createElement('a');
     a.textContent = `a${i}`;
     a.href= `a${i}.html`;
     const img = document.createElement('img');
-    const imgbox = ['/img/bookmark.png','/img/message.png'];
-    img.src = imgbox[i-1];
-    const fragment = document.createDocumentFragment();
-    {
-      fragment.appendChild(li).appendChild(a).insertBefore(img, a.firstChild);
-    }
-    ul.appendChild(fragment)
+    img.src = `/img/${imgbox[i-1]}`;
```

Comment:
[LGTM]

We have confirmed that the template literal has been modified.

Reply:
Thank you for confirming.